main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       knob
+//  action 1:   rotates clockwise
+knob.rotateCw();
+//  action 2:   rotates counterclockwise
+knob.rotateAcw();
+//  action 3:  when pushed, triggers a switch
+knob.switch();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       monitor
+//  action 1:   turns on/off
+monitor.switch();
+//  action 2:   adjusts brightness
+monitor.brightness();
+//  action 3:   adjusts refresh rate
+monitor.refreshRate();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       fork
+//  action 1:   stabs
+fork.stab();
+//  action 2:   bends
+fork.bend();
+//  action 3:   melts
+fork.melt();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       mixer
+//  action 1:   adjusts channel  volume with a fader
+mixer.faderVolume();
+//  action 2:   enables EQ
+mixer.eqSwitch();
+//  action 3:   adjusts compressor trheshold
+mixer.compThreshold(); 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       car
+//  action 1:   turns
+car.turn();
+//  action 2:   accelerates
+car.accelerate();
+//  action 3:   ejects the passenger seat
+car.ejectoSeatoCuz(); 

```

Comment:
lol but no
your plain english expression is `ejects the passenger seat`
your pseudo code should say `ejectPassengerSeat();`
- [ ] revise